exla/README.md

```diff
@@ -35,6 +35,17 @@ If running on Windows, you will also need:
 
 The first compilation will take a long time, as it needs to compile parts of Tensorflow + XLA. Subsequent commands should be much faster.
 
+#### Installation troubleshooting
+
+  * Bazel
+    * Use `bazel --version` to check your Bazel version, make sure you are using v3.1-v3.7.
+    * It can be installed with `asdf`:
+      * asdf plugin-add bazel
+      * asdf install bazel 3.1.0
+      * asdf global bazel 3.1.0
```

Comment:
I would also add that most of the binaries are available on Github: https://github.com/bazelbuild/bazel/releases

Reply:
Done!